src/main/scala/cromwell/backend/model/TaskStatus.scala

```diff
@@ -1,21 +1,24 @@
 package cromwell.backend.model
 
-import cromwell.backend.model.Status.Status
+import cromwell.caching.ExecutionHash
+import wdl4s.values.WdlValue
 
-/**
-  * Represents status of an execution.
-  */
-object Status extends Enumeration {
-  type Status = Value
-  val Created, Running, Succeeded, Failed, Canceled = Value
-}
+import scala.concurrent.Future
 
-/**
-  * Defines a task intermediate status.
-  */
-case class TaskStatus(status: Status) extends ExecutionEvent
 
-/**
-  * Defines a task final status with the resulting data.
-  */
-case class TaskFinalStatus(status: Status, result: ExecutionResult) extends ExecutionEvent
\ No newline at end of file
+sealed trait TaskStatus extends ExecutionEvent
+
+sealed trait IntermediateTaskStatus extends TaskStatus
+final case class CreatedTaskStatus(stdout: String, stderr: String) extends IntermediateTaskStatus
```

Comment:
They're here so they can be sent back to the call actor, which then forwards them to be persisted by the WorkflowActor.  This is done to be able to get references to backends out of the various metadata endpoints.  There was some yukky BackendCall#bindCall stuff going on to reconstruct this information after the call completed; it's a lot easier to do it here when we have the information in hand.